rss/serializers/atom_1_0.py

```diff
@@ -1,15 +1,81 @@
 """Serializer for Atom 1.0"""
 
-from flask.json import jsonify
+from feedgen.feed import FeedGenerator
+from flask import url_for
 from typing import Tuple, Optional
 from arxiv import status
 from rss.serializers.serializer import Serializer
+from rss.serializers.atom_extensions import ArxivExtension, ArxivEntryExtension
 from elasticsearch_dsl.response import *
 
+import datetime
+import pytz
+
 
 class Atom_1_0(Serializer):
 
     def get_xml(self, response: Response) -> Tuple[Optional[dict], int]:
-        data = jsonify({'version': "atom 1.0"})
+
+        fg = FeedGenerator()
+        fg.register_extension("arxiv", ArxivExtension, ArxivEntryExtension, rss=False)
+        fg.id("http://arxiv.org/rss/version=atom_1.0")
+        archive = response.hits[0]["primary_classification"]["archive"]
+        fg.title(archive["id"] + " updates on arXiv.org")
+        fg.link(href='http://arxiv.org/rss/version=atom_1.0', rel='self', type='application/atom+xml')
+        fg.updated(datetime.datetime.utcnow().replace(tzinfo=pytz.utc))
+
+        # TODO - Try to remove generator element?  This doesn't work - code ignores "None"
+        # fg.generator(None)
+        # TODO - We don't currently set "subtitle", but could do it like this
+        # fg.subtitle(f"{archive['name']} ({archive['id']}) updates on the arXiv.org e-print archive")
```

Comment:
Not being much of an RSS user any longer, and never was on arxiv, I don't have a strong preference - could be worth making an issue to get feedback from a wider group.